SimpleKanbanBoard/SimpleKanbanBoard.csproj

```diff
@@ -12,11 +12,13 @@
     <PackageReference Include="Blazorise" Version="1.0.6" />
     <PackageReference Include="Blazorise.Bootstrap" Version="1.0.6" />
     <PackageReference Include="Blazorise.Icons.FontAwesome" Version="1.0.6" />
+    <PackageReference Include="Blazorise.Sidebar" Version="1.0.6" />

```

Comment:
Don't use the Sidebar extension. It is obsolete. Use the Bar component https://blazorise.com/docs/components/bar